debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -773,7 +774,7 @@ public static GtidNewChannelPosition parse(String value, String defaultValue) {
 
     public static final Field ROW_COUNT_FOR_STREAMING_RESULT_SETS = Field.create("min.row.count.to.stream.results")
             .withDisplayName("Stream result set of size")
-            .withType(Type.LONG)
+            .withType(Type.INT)
```

Comment:
After adding this missing field to the `ConfigDef`/`ALL_FIELDS` the unit tests below failed because the default value is an INTEGER but the field was declared as `Type.LONG`.

Is this even a bug in `#withDefault()` working with only Object and not respecting/converting to the defined type?

Reply:
@gunnarmorling :arrow_up: